### ISSUE TITLE
PRandom Photo Screensaver as choice of screen saver (both "Random" and "PRandom" are available)

### ISSUE THREAD

**Issue by dansyl1** [marijnkampf/Random-Photo-Screensaver#129]
![PRandom Photo Screensaver](https://user-images.githubusercontent.com/67522741/85932686-2ac23100-b89c-11ea-8d20-9f0f27d444c1.jpg)
Windows 7 64bit, French. I installed and did not select to configure nor set as default, rebooted and then selected Random Photo Screensaver as screen saver, using 1 minute delay. It worked fined. I then went back and clicked "Parameters" to change the Metadata displayed. I now have two choices of sceensaver and they never start after the 1-minute delay. 

Also, if i right click the sceen and select "Personalize" I get a screen saying no screensaver is configured. I checked earlier (when it was working) and it specified Random Photo Screensaver was set up.

Tried to select No screen saver, clicking OK and going back to select Random Photo Screensaver, issue still present.

**Comment by dansyl1** [comment]
The following web site states that a scr filename cannot have a: https://www.sevenforums.com/tutorials/2109-screen-saver-change.html. I rename "Random Photo Screensaver.scr" to Random_Photo_Screensaver.scr and the drop down list in Windows now shows Random_Photo_Screensaver consistently (no more name corruption like I had before), but the name is not the intended one.

According to https://stackoverflow.com/questions/1341575/c-retrieve-names-of-installed-screen-savers, a screensaver display name is the first resource string in the .scr file; for Windows built-in screensavers it is in a .scr.mui file: .

_Basically, the display name of the screensaver is the first resource string in the .scr file. Note that for some screensavers (e.g. Windows built-in screensavers), the localized resources are not in the main .scr file but in a .scr.mui file in a culture-specific subdirectory. You don't have to worry about it, because LoadString knows where to find the adequate resource._

Fix would thus be to distribute the .scr with no space in filename and make the first resource string be "Random Photo Screensaver", but I do not know how this can be done in the project.

**Comment by dansyl1** [comment]
Did not intend to close the issue...
